### DISCUSSION TITLE
Error while downloading RIS results from ASReview Lab

### DISCUSSION THREAD

**Discussion by numland** [asreview/asreview#1094] [Troubleshooting (report errors in the "Issues" tracker)] [answered]
When I try to download the RIS results from my current project I get an error message: "The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()"
Other download options like Excel work without issues. 

Any suggestions on how to export to RIS?'

**Comment by Rensvandeschoot** [comment]
Did you use the latest version of ASReview LAB (0.19.2)?

**Reply by numland** [reply]
Yes I did

**Reply by numland** [reply]
Well, I started the project on a later version. But in the course of trying to fix the problem I updated to 0.19.2

**Reply by Rensvandeschoot** [reply]
Ok, and did you create this RIS file in Zotero or Endnote?

**Reply by numland** [reply]
For the import I created a RIS file from Citavi. The error pops up, when I try to export.

**Reply by Rensvandeschoot** [reply]
Can you export the project and send it to asreview@uu.nl?

**Reply by numland** [reply]
I sent you the project

**Comment by Rensvandeschoot** [comment, marked as answer]
A fix is available. Please upgrade to 0.19.3. This release fixes a bug with RIS exports reported on the discussion platform. The bug shows up on the export when RIS files with secondary_authors are uploaded.

**Reply by numland** [reply]
I just installed the update and everything works like a charm.
Great! Thanks a lot!
